h5py/_hl/filters.py

```diff
@@ -146,7 +150,7 @@ def __init__(self, level=DEFAULT_GZIP):
 
 def fill_dcpl(plist, shape, dtype, chunks, compression, compression_opts,
               shuffle, fletcher32, maxshape, scaleoffset, external,
-              allow_unknown_filter=False):
+              allow_unknown_filter=False, fill_time=None):
```

Comment:
This is better, but I'd still prefer we make it keyword only:
```suggestion
              allow_unknown_filter=False, *, fill_time=None):
```